.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,65 @@
+#For more details
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
+# https://docs.gitea.com/1.19/usage/issue-pull-request-templates
+#Be sure to check the docs
+#
+#Format
+#- type: | markdown | textarea | input | checkboxes |
+#  id:   | just an id, no actual relevance |
+#  attributes:
+#    label:       | Box title |
+#    description: | Box small text - Description |
+#
+name: Bug Report
+#title: "[Bug]: " | if you want to have a have it automatically say [Bug] when they start the form
+description: Create a report to help us improve. If this doesn't look right
+#labels: [Bug] | Auto labels the post, as we do not use this, commenting out, purely as reference
+
+body:
+- type: markdown
+  attributes:
+    value: "**IMPORTANT** <br> If your issue is an ingame exploit that allows players to gain an unfair advantage, or an issue which could affect server stability or security, please file it at https://www.paradisestation.org/forum/179-exploit-reports/ so that it is safe from prying eyes"
+- type: textarea
+  id: Issue
+  attributes:
+    label: Issue Description
+    description: What is the problem?
+- type: textarea
+  id: What-Happened
+  attributes:
+    label: What did you expect to happen
+    description: Why do you think this is an issue?
+- type: textarea
+  id: What-Instead
+  attributes:
+    label: What happened instead
+    description: How is what happened different from what you expected?
+- type: textarea
+  id: Why
+  attributes:
+    label: Why is this bad/What are the consequences
+    description: Why do you think this is an important issue?
+- type: textarea
+  id: How
+  attributes:
+    label: Steps to reproduce the problem
+    description: The most important section. Review everything you did leading up to causing the issue.
+- type: textarea
+  id: When
+  attributes:
+    label: When did the problem start happening
+    description: If your report is about something that used to work but no longer does, when was the last time you remember it working?
+- type: textarea
+  id: Extra-Info
+  attributes:
+    label: Extra information
+    description: If your report is about something that used to work but no longer does, when was the last time you remember it working?
+- type: input
+  id: Round-ID
+  attributes:
+    label: Round-ID
```

Comment:
I'd lump this in with the extra information, maybe with an example text talking about it?

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/66401072/97e9a6c7-38f1-4cc7-84a3-b52ae9c0db80)
Better?